pybossa/model/project.py

```diff
@@ -199,4 +199,16 @@ def set_project_users(self, users):
     def get_project_users(self):
         return self.info.get('project_users', [])
 
+    def get_quiz_pass_fail(self, right_count, wrong_count):
+        quiz = self.info.get('quiz')
+        if not quiz:
+            raise Exception('Quiz is not configured for this project.')
+        if right_count >= quiz.get('correct_answers_to_pass'):
```

Comment:
can "correct_answers_to_pass" be "pass_count" instead as correct answer is implicit to pass?

Reply:
`pass_count` sounds like the number of times they passed the quiz. How about I change `correct_answers_to_pass` to just `pass` and `questions_per_quiz` to just `questions`?

Reply:
looks fine to me.